tests/lava/lib/peripherals/dvs/test_prophesee.py

```diff
@@ -25,15 +25,23 @@
 from lava.magma.core.run_configs import Loihi2SimCfg
 from lava.magma.core.run_conditions import RunSteps, RunContinuous
 
-from lava.lib.peripherals.dvs.prophesee import (
-    PropheseeCamera,
-    PyPropheseeCameraModel,
-)
 from lava.lib.peripherals.dvs.transformation import Compose, Downsample
 from metavision_core.utils import get_sample
-from metavision_core.event_io import RawReader, EventDatReader
+from metavision_core.event_io import (
+    RawReader, EventDatReader, EventsIterator)
 from metavision_sdk_cv import ActivityNoiseFilterAlgorithm
 
+USE_EVENTSITERATOR = True
```

Comment:
There need to be unit tests for both the ProcessModel using the RawReader as well as the one using the EventsIterator. A global switch for the unit tests that would have to be changed manually before running the tests does not work here.

Please make sure that there are separate tests for the different ProcessModel implementations and that the correct ProcessModels are imported for those tests only.

Reply:
This comment has not been addressed, yet. As it is, only the ProcessModel with the EventsIterator implementation is being tested by the unit tests. Please remove this global flag and write dedicated tests for the PropheseeCamera Process with its different ProcessModels (i.e., RawReader and EventsIterator).

If there is something blocking you, please do not hesitate to reach out and discuss that with me.